core/controllers/ingest.py

```diff
@@ -394,6 +398,8 @@ def populate_db(
     mappings: tuple[DataMapping],
     excel_file: FileStorage,
     load_mapping: dict[str, Callable],
+    submitting_account_id=None,
+    submitting_user_email=None,
```

Comment:
Could you add type hints to these please? Both are `str | None`. If you can cascade these types down as needed to other functions too, that'd be grand, thanks.